integration-tests/pom.xml

```diff
@@ -215,6 +215,7 @@
                 <module>hibernate-reactive-mariadb</module>
                 <module>hibernate-reactive-mysql</module>
                 <module>hibernate-reactive-mysql-agroal-flyway</module>
+                <module>hibernate-reactive-oracle</module>
```

Comment:
This makes the IT module part of the build, but in order for native tests to run on CI we also need to update `native-tests.json`.

See:

https://github.com/quarkusio/quarkus/commit/e1f94a99c03315bfee05b78ad016df7a311d13ee#diff-307c72aded80fc768e1f20cfa0bed2320c3fe4bb815a6996db89d986c972d845L47-L49

Don't forget to add 5 minutes to the timeout, otherwise this added test might make the whole build time out :)